persistence/src/main/java/org/mskcc/cmo/metadb/persistence/MetaDbSampleRepository.java

```diff
@@ -72,4 +72,8 @@ MetaDbSample findMetaDbSampleByRequestAndIgoId(@Param("reqId") String reqId,
             + "RETURN sm"
     )
     List<SampleMetadata> findSampleMetadataListByCmoPatientId(@Param("cmoPatientId") String cmoPatientId);
+    
+    @Query("MATCH (sm:SampleMetadata {cmoInfoIgoId: $igoSampleId})"
+            + "RETURN sm")
+    List<SampleMetadata> getSampleMetadataHistory(@Param("igoSampleId") String igoSampleId);
```

Comment:
did you confirm that this query works as we expect it to? like just executing the query directly in the neo4j browser?